src/ui/components/SupportForm.tsx

```diff
@@ -0,0 +1,216 @@
+          </div>
+        </div>
+      </div>
+    );
+  } else {
+    const closeIfEmpty = () => {
+      if (!text) {
+        dispatch(setShowSupportForm(false));
+      }
+    };
+
+    const submit = async () => {
+      if (text) {
+        try {
+          const result = await fetch("/api/feedback", {
+            method: "POST",
+            headers: {
+              "Content-Type": "application/json",
+              Accept: "application/json",
+            },
+            // FormCarry does not support nested values
+            body: JSON.stringify({
+              date: new Date(),
+              recordingId,
+              sessionId,
+              text,
+              url: window.location.href,
+              userAgent: navigator.userAgent,
+              userEmail: currentUserInfo?.email,
+              userId: currentUserInfo?.id,
+              userName: currentUserInfo?.name,
+            }),
+          });
+
+          if (result.status >= 400) {
+            const text = await result.text();
+
+            throw Error(text);
+          }
+
+          setSubmitStatus("success");
+        } catch (error) {
+          setSubmitStatus("failure");
+        } finally {
+          setText("");
+        }
+      }
+    };
+
+    let confirmationMessage = null;
+    switch (submitStatus) {
+      case "failure":
+        confirmationMessage = "Something went wrong. Please reload the page and try again.";
+        break;
+      case "success":
+        if (currentUserInfo) {
+          confirmationMessage = `Thank you for your note.' We'll get back to you at ${currentUserInfo.email} as son as possible.`;
+        } else {
+          confirmationMessage = "Thank you for your note.";
+        }
+        break;
+    }
+
+    return (
+      <div className={styles.Background}>
+        <div className={styles.Modal} ref={modalRef}>
+          <div className={styles.Header}>
+            <div>Support</div>
+            <button className={styles.CloseButton} onClick={confirmClose}>
+              <Icon className={styles.CloseButtonIcon} type="close" />
+            </button>
+          </div>
+          {confirmationMessage ? (
+            <div className={styles.ConfirmationMessage}>{confirmationMessage}</div>
+          ) : (
+            <>
+              <textarea
+                autoFocus
+                className={styles.TextArea}
+                onChange={event => setText(event.target.value)}
+                placeholder="Bugs, suggestions, questions are all welcome!"
+                value={text}
+              />
+              <div className={styles.FormActions}>
+                {/* TODO [FE-1894] Add screenshot function (eventually)
+            <SecondaryButton
+              className={styles.ScreenshotButton}
+              color="gray"
+              onClick={takeScreenshot}
+            >
+              <Icon className={styles.ScreenshotButtonIcon} type="screenshot" />
+            </SecondaryButton>
+            */}
```

Comment:
I decided to cut scope for this feature. I don't think it's worth doing now (if at all) and we can follow up on Linear.